thirdparty/chart-tp-api-svc/participant-list.svc.yaml

```diff
@@ -0,0 +1,213 @@
+        OAUTH2_TOKEN_ISS: http://$auth_host:$auth_port$auth_token_iss_path
+        P12_PASS_PHRASE: SOME_S3C4R3_P@SS
+        PTA_ROLE: Application/PTA
+    user_config.json:
+      ADVANCED_FEATURES_ENABLED: true
+      # Endpoint of FSPIOP API... I think
+      CALLBACK_ENDPOINT: http://beta.moja-lab.live/api/fspiop
+      CALLBACK_RESOURCE_ENDPOINTS:
+        enabled: true
+        endpoints:
+        - endpoint: http://moja-account-lookup-service.demo
+          method: put
+          path: /parties/{Type}/{ID}
+        - endpoint: http://moja-quoting-service.demo
+          method: put
+          path: /quotes/{ID}
+        - endpoint: http://moja-ml-api-adapter-service.demo
+          method: put
+          path: /transfers/{ID}
+      CALLBACK_TIMEOUT: 10000
+      DEFAULT_USER_FSPID: userdfsp
+      ENDPOINTS_DFSP_WISE:
+        dfsps:
+          userdfsp:
+            defaultEndpoint: http://mojaloop-sim-switch-scheme-adapter:4000
+            endpoints: []
+          userdfsp2:
+            defaultEndpoint: http://scheme-adapter2:4000
+            endpoints: []
+      FSPID: swich
+      HUB_ONLY_MODE: false
+      ILP_SECRET: secret
+      INBOUND_MUTUAL_TLS_ENABLED: false
+      JWS_SIGN: false
+      JWS_SIGN_PUT_PARTIES: false
+      OUTBOUND_MUTUAL_TLS_ENABLED: false
+      SEND_CALLBACK_ENABLE: true
+      TRANSFERS_VALIDATION_CONDITION: true
+      TRANSFERS_VALIDATION_ILP_PACKET: true
+      TRANSFERS_VALIDATION_WITH_PREVIOUS_QUOTES: true
+      VALIDATE_INBOUND_JWS: false
+      VALIDATE_INBOUND_PUT_PARTIES_JWS: false
+      VERSION: 1
+      VERSIONING_SUPPORT_ENABLE: true
+  dependencies:
+    adminApiAuthentication:
+      embeddedCertificate: password
+      host: localhost
+      paths:
+        issuer: /api/oauth2/token
+        tokenIss: /api/oauth2/token
+      port: "5050"
+    connectionManagerBackend:
+      enabled: false
+      host: $release_name-connection-manager-backend
+      port: "5061"
+    keycloak:
+      embeddedCertificate: '-----BEGIN PUBLIC KEY-----\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDda1T6cyWogfG/xlUb3+gWZI0jsnlSGbf3PF89NCgS+n6HUJ0A2Ezmu/n9Gfkm8Rjyst914Vq9ebsueXHIc+Ad76+q3MASF1hRUTfnvH/v6rDguPvI6Jb/M9jGn5b2hglcg3B9Y7Tgv70bcV7oCweP68CPbIrTWjR9gyjmSHfYhwIDAQAB\n-----END
+        PUBLIC KEY-----'
+      enabled: false
+      host: $release_name-keycloak-http
+      paths:
+        issuer: /auth/realms/testingtoolkit/protocol/openid-connect/token
+        tokenIss: /auth/realms/testingtoolkit
+      port: "7080"
+    mongodb:
+      enabled: false
+      host: $release_name-ttk-mongodb
+      port: "27017"
+  enabled: true
+  extraEnvironments: {}
+  global: {}
+  image:
+    tag: v13.1.0
+    # Load some config from ml-oss-sandbox master branch
+    # hopefully soon we will have a better way to load
+    # apis into the TTK
+    command: >
+      ["sh", "-c", 
+        "BRANCH_NAME=master;
+        REPO_URL=https://github.com/vessels-tech/ml-oss-sandbox/archive/master.zip;
```

Comment:
It loads the `/services/{Type}` API, since we can't specify ttk api definitions inline.

From the top of that file:
># The actual config for the /services/{Type} API comes from an external repo: 
># https://github.com/vessels-tech/ml-oss-sandbox This is a workaround because 
># we can't specify inline TTK configs from within this values file.